tests/python/direct/test_tutorial.py

```diff
@@ -0,0 +1,138 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025-present NVIDIA CORPORATION & AFFILIATES.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# Owner(s): ["module: nvfuser"]
+
+import torch
+from nvfuser_direct import FusionDefinition, ParallelType
+
+verbose_ = False
+
+
+def test_tutorial_memcpy():
+    # First, we define a fusion. A common pattern is:
+    # - Declare a Fusion, which works as a container of expressions
+    # - Setup inputs. Utility routines such as makeSymbolicTensor
```

Comment:
Looks like the comments need to be adjusted as well

Reply:
Updated